packages/base/source/server/events.js

```diff
@@ -5,7 +5,7 @@ Space.messaging.define(Space.messaging.Event, 'Todos', {
   },
 
   TodoCreated: {
-    id: Guid,
+    todoId: Guid,
```

Comment:
Yes, that is the main problem. It will go away once we have documentation. But this suggestion, `aggregateId` instead of `sourceId`and `targetId`is so explicit so it should be considered. Most of the people that will try to see what Space is will not be experts in CQRS at first. But it can also be processId. So strike this. :)